lesson_14/class_student.py

```diff
@@ -0,0 +1,16 @@
+class Student:
+
```

Comment:
Where is class fields? You need to create fields for further properties assignment.

Reply:
What you mean is that I need to add the following:
name: str
last_name: str
age: int
average_grade: float  ?